Jenkinsfile_CNP

```diff
@@ -9,9 +9,52 @@ def component = "manage-case-assignment"
 
 GradleBuilder builder = new GradleBuilder(this, product)
 
+def secrets = [
+  'ccd-${env}': [
+    secret('ccd-caseworker-autotest-email', 'CCD_CASEWORKER_AUTOTEST_EMAIL'),
+    secret('ccd-caseworker-autotest-password', 'CCD_CASEWORKER_AUTOTEST_PASSWORD'),
+    secret('ccd-importer-autotest-email', 'CCD_IMPORT_AUTOTEST_EMAIL'),
+    secret('ccd-importer-autotest-password', 'CCD_IMPORT_AUTOTEST_PASSWORD'),
+    secret('ccd-api-gateway-oauth2-client-secret', 'OAUTH2_CLIENT_SECRET')
+  ],
+  's2s-${env}': [
+    secret('microservicekey-ccd-gw', 'BEFTA_S2S_CLIENT_SECRET')
+  ]
+]
+
+static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
+  [$class     : 'AzureKeyVaultSecret',
+    secretType : 'Secret',
+    name       : secretName,
+    version    : '',
+    envVariable: envVar
+  ]
+}
+
+def vaultOverrides = [
+  'preview' : 'aat',
+  'spreview': 'saat'
```

Comment:
spreview /saat is not available  anymore. Please remove.

Reply:
👍 and I will add a comment to explain why CCD vault is being referenced.